### PR TITLE
Truncate long descriptions for the Pagerduty handler, and log failures

### PR DESCRIPTION
**Pull request by vulpine**
This will cap a description to 1024 characters, which is the maximum length the Pagerduty handler supports. It will log any problems in raising the alert.

Manually tested this by creating an alert with a sufficiently long description; it paged and truncated in PD.

### CODE REVIEW

File: files/pagerduty.rb [hunk 1]
```diff
@@ -18,12 +18,18 @@ def api_key
   def trigger_incident
     return false unless api_key
     require 'redphone/pagerduty'
-    Redphone::Pagerduty.trigger_incident(
+    response = Redphone::Pagerduty.trigger_incident(
       :service_key  => api_key,
       :incident_key => incident_key,
-      :description  => description,
+      :description  => description(1024),
```
Comment (solarkennedy) [inline files/pagerduty.rb]: The docs only hint at this particular field having a limit:

Comment (solarkennedy) [inline files/pagerduty.rb]: https://developer.pagerduty.com/documentation/integration/events/trigger

So that is good

File: files/pagerduty.rb [hunk 2]
```diff
@@ -18,12 +18,18 @@ def api_key
...
       :details      => full_description_hash
-    )['status'] == 'success'
+    )['status']
+    if response != 'success'
```
Comment (solarkennedy) [inline files/pagerduty.rb]: Can you invert the logic so that if the response == success => true, but anything else is false?

File: README.md [hunk 3]
```diff
@@ -179,6 +179,19 @@ sensu_handlers::teams:
 * *`notification_email: 'operations@localhost'`* - If set, the handler will send emails for every event to this address. If ommited it will send no emails. You can send the email to multiple destinations by using comma separated list (like any email client)
 * *`project: OPS`* - Used by the JIRA handler. If a event comes in that has `ticket=>true`, the jira handler will open a ticket on this project. There no default for this parameter. Special considerations have to be made for the JIRA project to enable auto-opening and auto-closing of tickets, see the docs on the jira handler.
 
+
+### Manually Invoking These Handlers
+
+You can manually invoke these handlers in order to test them, ensuring that (for example)
```
Comment (solarkennedy) [inline README.md]: nice

File: README.md [hunk 4]
```diff
@@ -179,6 +179,19 @@ sensu_handlers::teams:
...
+
+### Manually Invoking These Handlers
+
+You can manually invoke these handlers in order to test them, ensuring that (for example) 
+a JIRA ticket is correctly raised. Simply pipe the Sensu alert in JSON into one of the 
+handlers, and it should parse it as if it were a fresh alert.
+
+```
+$ grep 'failed' /var/log/sensu/sensu-server.log  | tail -n 1 | jq .event > last_failed_event.json
+$ cat last_failed_event | sudo ruby jira.rb
```
Comment (solarkennedy) [inline README.md]: sudo -u sensu

**Comment by vulpine** [comment]
Updated per solarkennedy's suggestions.

**Comment by solarkennedy** [comment]
Ship!

**Pull request merged**